From c75855df6d18313fcbb985f7fa5f796a4163384c Mon Sep 17 00:00:00 2001
From: Anurag Chouhan <achouhan@codeaurora.org>
Date: Fri, 21 Jul 2017 17:24:36 +0530
Subject: [PATCH 3/3] wcnss_service: Change WCNSS_qcom_cfg.ini file path

Add changes to support treble feature on pronto, change
WCNSS_qcom_cfg.ini file path from /system to /vendor

Change-Id: I7082642d1ac34f7eb5c07edec34c3be05494d2c4
CRs-fixed: 2082792
Signed-off-by: sub77 <sub77@ymail.com>
---
 qcwcn/wcnss-service/wcnss_service.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/qcwcn/wcnss-service/wcnss_service.c b/qcwcn/wcnss-service/wcnss_service.c
index 573368d..37bea9f 100644
--- a/qcwcn/wcnss-service/wcnss_service.c
+++ b/qcwcn/wcnss-service/wcnss_service.c
@@ -70,7 +70,7 @@ ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #define WCNSS_DEVICE    "/dev/wcnss_wlan"
 #define WCNSS_CTRL      "/dev/wcnss_ctrl"
 #define WLAN_INI_FILE_DEST   "/data/misc/wifi/WCNSS_qcom_cfg.ini"
-#define WLAN_INI_FILE_SOURCE "/system/etc/wifi/WCNSS_qcom_cfg.ini"
+#define WLAN_INI_FILE_SOURCE "/vendor/etc/wifi/WCNSS_qcom_cfg.ini"
 #define WCNSS_HAS_CAL_DATA\
 		"/sys/module/wcnsscore/parameters/has_calibrated_data"
 #define WLAN_DRIVER_ATH_DEFAULT_VAL "0"
-- 
2.1.2

